BenchmarkSample/benchmark/build.gradle

```diff
@@ -8,6 +8,15 @@ android {
     defaultConfig {
         minSdkVersion 14
         targetSdkVersion 29
+
+        // Change this to enable profiling, can be one of:
```

Comment:
Set this argument to capture profiling information, instead of measuring performance. Can be one of:

Reply:
Done